package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "mr-social-client",
+  "name": "hubs",
```

Comment:
good fix! while you're at it, would be nice to add a `description` 👍

Reply:
That's true, I didn't go very overboard because I don't think we really have a use case for someone pulling down our package from NPM right now. Do you know whether there are other reasons we should care about having accurate metadata in package.json?